src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const RENT_CAR = 40;
```

Comment:
```suggestion
  const DAILY_PRICE = 40;
```
The name is not fully accurate